packages/amplication-server/src/core/entity/entity.service.ts

```diff
@@ -758,12 +758,12 @@ export class EntityService {
     return entities;
   }
 
-  async createDefaultEntities(
+  async createDefaultUserEntity(
     resourceId: string,
     user: User
   ): Promise<Entity[]> {
     return await Promise.all(
-      DEFAULT_ENTITIES.map(async (entity) => {
+      DEFAULT_USER_ENTITY.map(async (entity) => {
```

Comment:
See my previous comment, if it's not an array, you can get rid of the .map() here.

Reply:
I suggest to keep it as an array for now for risk management reasons